proto/ibc/core/channel/v2/query.proto

```diff
@@ -5,132 +5,60 @@ package ibc.core.channel.v2;
 option go_package = "github.com/cosmos/ibc-go/v9/modules/core/04-channel/v2/types";
 
 import "cosmos/base/query/v1beta1/pagination.proto";
-import "ibc/core/channel/v2/channel.proto";
 import "ibc/core/channel/v2/genesis.proto";
 import "ibc/core/client/v1/client.proto";
 import "google/api/annotations.proto";
-import "google/protobuf/any.proto";
 import "gogoproto/gogo.proto";
 
 // Query provides defines the gRPC querier service
 service Query {
-  // Channel queries the counterparty of an IBC client.
-  rpc Channel(QueryChannelRequest) returns (QueryChannelResponse) {
-    option (google.api.http).get = "/ibc/core/channel/v2/channels/{channel_id}";
-  }
-
-  // ChannelClientState queries for the client state for the channel associated
-  // with the provided channel identifiers.
-  rpc ChannelClientState(QueryChannelClientStateRequest) returns (QueryChannelClientStateResponse) {
-    option (google.api.http).get = "/ibc/core/channel/v2/channels/{channel_id}/client_state";
-  }
-
-  // ChannelConsensusState queries for the consensus state for the channel associated
-  // with the provided channel identifiers.
-  rpc ChannelConsensusState(QueryChannelConsensusStateRequest) returns (QueryChannelConsensusStateResponse) {
-    option (google.api.http).get = "/ibc/core/channel/v2/channels/{channel_id}/consensus_state";
-  }
-
   // NextSequenceSend returns the next send sequence for a given channel.
   rpc NextSequenceSend(QueryNextSequenceSendRequest) returns (QueryNextSequenceSendResponse) {
-    option (google.api.http).get = "/ibc/core/channel/v2/channels/{channel_id}/next_sequence_send";
+    option (google.api.http).get = "/ibc/core/channel/v2/clients/{client_id}/next_sequence_send";
```

Comment:
There rpc urls are very ugly since they still contain channel/v2. Should we pull this whole thing back out to its own package `packetserver`? 🫣

Reply:
I don't mind the urls but up to you